docker/vernemq/Dockerfile

```diff
@@ -47,7 +47,7 @@ EXPOSE 1883 8883 8080 44053 4369 8888 \
 
 VOLUME ["/vernemq/log", "/vernemq/data", "/vernemq/etc"]
 
-HEALTHCHECK CMD vernemq ping | grep -q pong
+HEALTHCHECK CMD curl -s -f http://localhost:8888/health || false
```

Comment:
Just a comment here: if we changed VerneMQ Dockerfile, we'll need to build and upload new image. I am not sure of the current CI covers the VerneMQ - if not, please add it.